### PR TITLE
Add password encryption for Symanted Endpoint Protection V2

### PR DESCRIPTION
**Pull request by ostolero**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/47923)

## Description
Client's received authentication error for passwords with special characters. added encryption to resolve the issue.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Tests/conf.json [hunk 1]
```diff
@@ -5244,7 +5244,8 @@
         "Recorded Future Test": "Issue 26741",
         "Git_Integration-Test": "Issue 37800",
         "RecordedFutureFeed - Test": "Issue 43923",
-        "RedLockTest": "Issue 24600"
+        "RedLockTest": "Issue 24600",
+        "SymantecEndpointProtection_Test": "Unable to connect to instance"
```
Comment (moishce) [inline Tests/conf.json:5248]: Please reopen the [issue](https://github.com/demisto/etc/issues/30157) of the Test-Failure and the reason why he failed

**Comment by ostolero** [comment]
The build is failing because no playbook tests are enabled (because our instance is down).
A force merge is needed.

File: Packs/SymantecEndpointProtection/ReleaseNotes/1_1_0.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Symantec Endpoint Protection v2
+- Fixed an issue where passwords with special characters received authentication error.
```
Comment (ShahafBenYakir) [inline Packs/SymantecEndpointProtection/ReleaseNotes/1_1_0.md:4]:
```suggestion
- Fixed an issue where passwords with special characters raised authentication error.
```

**Review by ShahafBenYakir — APPROVED** [review]
Small RN comment

**Comment by ShahafBenYakir** [comment]
Merging as integration was tested on the customer side and succeeded.

**Pull request merged**